src/main/java/bio/terra/app/controller/SnapshotsApiController.java

```diff
@@ -219,6 +219,16 @@ public ResponseEntity<SnapshotModel> retrieveSnapshot(
     return ResponseEntity.ok(snapshotModel);
   }
 
+  @Override
+  public ResponseEntity<SnapshotSummaryModel> retrieveSnapshotSummary(UUID id) {
+    logger.info("Verifying user access");
+    AuthenticatedUserRequest authenticatedInfo = getAuthenticatedInfo();
+    snapshotService.verifySnapshotListable(id, authenticatedInfo);
+    logger.info("Retrieving snapshot summary");
```

Comment:
I get having this for testing but I wonder if this is too verbose (given how much logging we do, I wonder if we should be doing less since it's pretty expensive)

Reply:
Might as well leave as debug statements (now I need to figure out the best way to turn on debug in prod :) )